.travis.yml

```diff
@@ -31,6 +31,15 @@ jobs:
         - make lint
         - make coverage
         - make validate-commit
+    - script:
+      - docker run -d --name bblfshd --privileged --volume $HOME/bblfsh-drivers/images:/var/lib/bblfshd bblfsh/bblfshd
```

Comment:
You need to cache only images directory not all `/var/lib/bblfshd`. Because bblfsh writes there some other stuff that changes after every run. It will cause recreating and uploading cache every time.

Reply:
Thanks, It was a copy&paste mistake. The volume was supposed to be `$HOME/bblfsh-drivers`, the cache dir already points to `$HOME/bblfsh-drivers/images`.